content/en/docs/concepts/cluster-administration/garbage-collection-container-images.md

```diff
@@ -1,13 +1,13 @@
 ---
 reviewers:
-title: Configuring kubelet Garbage Collection
+title: Garbage collection for container images
 content_type: concept
 weight: 70
 ---
 
 <!-- overview -->
 
-Garbage collection is a helpful function of kubelet that will clean up unused images and unused containers. Kubelet will perform garbage collection for containers every minute and garbage collection for images every five minutes.
+Garbage collection is a helpful function of kubelet that will clean up unused [images](https://kubernetes.io/docs/concepts/containers/#container-images) and unused [containers](https://kubernetes.io/docs/concepts/containers/). Kubelet will perform garbage collection for containers every minute and garbage collection for images every five minutes.
```

Comment:
Hi @Kartik494 .
Do you want to link to the containers page from the GC page or the other way around?
Also, I suggest to write the like:
```
[containers](/docs/concepts/containers/)
```

If you change the name of the file (not sure if required), you will want to add a redirect for the original page?
See `website/static/_redirects`.

Reply:
Hi @kbhawkey i agreed i will be changing the title only and change the name of file to the original one  and i will be updated the link as you mentioned .
Thanks